workflow/envs/fix_af.yaml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - pysam=0.17.0
```

Comment:
if fix_af is using common container pysam version should be changed to 0.18.0

Reply:
done